include/crm/crm.h

```diff
@@ -51,7 +51,7 @@ extern "C" {
  * >=3.0.13: Fail counts include operation name and interval
  * >=3.2.0:  DC supports PCMK_LRM_OP_INVALID and PCMK_LRM_OP_NOT_CONNECTED
  */
-#  define CRM_FEATURE_SET		"3.6.2"
+#  define CRM_FEATURE_SET		"3.6.3"
```

Comment:
To help future us, add something to the commit message about why it's bumping :)

Reply:
Done.